website/src/components/Showcase/AppsList.js

```diff
@@ -0,0 +1,88 @@
+const cardList = [
```

Comment:
This card list looks better than the one you have defined in JSON. 🤔

Reply:
Yes, the JS file is nice but spoke with @asafkorem about this and we decided to use a JSON file as it is only data here and easier to locate in the project. But I cleaned up the image url's as you suggested with the @site reference + location of file.